packages/common/__test__/common-basic.test.ts

```diff
@@ -230,7 +235,7 @@ describe('Basic CRUD', () => {
 
   describe('Typeorm Error Handling', () => {
     test('TypeORM errors should be returned verbosely when facing them', async () => {
-      const payload = { idNumber: 1 };
+      const payload = { idNumber: 1, createdAt: Date.now(), updatedAt: Date.now() };
```

Comment:
I believe the the form creation test cases need to be updated as well, because for ezbackend you can submit either POST data or multipart data (Which automatically handles file uploading)